package.json

```diff
@@ -27,18 +27,18 @@
         "precommit": "cd demo && bash ./scripts/buildBlank.sh && cd .. && yarn lint && yarn prettier"
     },
     "dependencies": {
```

Comment:
@bkarambe This package will need to be updated to contain latest versions of all deps for each template.